flashcards/backend/decks/models.py

```diff
@@ -20,3 +43,46 @@ def __str__(self):
     class Meta:
         verbose_name = "category"
         verbose_name_plural = "categories"
+
+
+class Deck(TimeStampModel):
+    """Model for representing deck data
+
+    :param image: deck image
+    :type image: file, optional
+    :param name: deck name
+    :type name: str
+    :param is_public: indicates if deck is public
+    :type is_public: bool, optional
+    :param price: deck price
+    :type price: Decimal, min_value=0.00
+    :param author: deck author
+    :type author: User
+    :param popularity: deck popularity
+    :type popularity: int, optional
+    :param rating: deck rating
+    :type rating: int, optional
+    :param is_active: indicates if deck is active
+    :type is_active: bool, optional
+    :param description: deck description
+    :type description: str
+    """
+
```

Comment:
We have django-stubs-ext. If you run mypy from inside the app mypy won't see the stubs and will thorow an error, besides type `str` for `name` is wrong because `name` has type `model.CharField` remove those typings and then run `docker compose exec backend mypy ./decks` or sth like that will show you that models are already typed